packages/engine/src/simulation/package/state/packages/behavior_execution/package.js

```diff
@@ -173,8 +173,8 @@ const run_task = (experiment, sim, task_message, group_state, group_context) =>
     for (var i_agent = 0; i_agent < n_agents_in_group; ++i_agent) {
         // TODO: Reuse `agent_state` objects. When using the old `agent_state`, the indices for behaviors are borked
         agent_state = group_state.get_agent(i_agent, null);
-        // Reuse `agent_ctx` objects.
-        agent_ctx = group_context.get_agent(i_agent, agent_ctx);
+        // TODO: Reuse `agent_ctx` objects. When using the old `agent_ctx`
+        agent_ctx = group_context.get_agent(i_agent, null);
```

Comment:
When we have proper testing in place, try reusing the context object again and see if it breaks anything. (Alternatively, fix the error with reusing the state object and see if the same error applies to the context object.)